inc/jetpack/class-storefront-jetpack.php

```diff
@@ -90,10 +90,7 @@ public function jetpack_infinite_scroll_loop() {
 		 *
 		 * @return void
 		 */
-		public function jetpack_infinite_scroll_wrapper_columns() {
-			add_action( 'storefront_jetpack_product_infinite_scroll_before', 'storefront_product_columns_wrapper' );
-			add_action( 'storefront_jetpack_product_infinite_scroll_after', 'storefront_product_columns_wrapper_close' );
-		}
+		public function jetpack_infinite_scroll_wrapper_columns() {}
```

Comment:
What's the deprecated feature here? I don't see a woo version number. 

@nielslange  can you provide more info about this change in the PR description?

If we do end up removing the body of this routine, should remove the whole function too?